features/definitions_transformations_annotations.feature

```diff
@@ -29,10 +29,10 @@ Feature: Step Arguments Transformations Annotations
       | --suite        | table_argument_transformation |
     Then it should pass with:
       """
-      ......
+      ............
 
-      3 scenarios (3 passed)
-      9 steps (9 passed)
+      4 scenarios (4 passed)
+      12 steps (12 passed)
```

Comment:
Might be just Monday morning, but I'm not following why the scenario count has changed for this feature and the row table argument transformations feature in the second commit?

Reply:
I rebased the code to include the changes I had added for the transformers with unicode characters and I included those changes in the existing commits but I made a weird split, I added the changes to the files in the fixtures in the first commit but only added the changes to the main feature file in the second commit 😬 . Hope this makes sense, if not I can try to rebase this again and split it better

Reply:
Makes sense, I think it's fine, just wanted to make sure there wasn't some unexpected change / I wasn't missing something.